### PR TITLE
Prompt user to finish exam when every question has been answered.

### LINKED ISSUE #198: Provide guidance when all questions are answered
Add extra instructions/links on the sidebar view once all questions are answered. E.g., "return to overview page (link) to finish test".

### PR DESCRIPTION
**Pull request by kevinwang**
Closes #198.

### CODE REVIEW

File: frontend/require/RetryExamTestSidebarView.html [hunk 1]
```diff
@@ -2,6 +2,12 @@
   <div class="panel-heading"><h3 class="panel-title"><a href="#ti/{{tid}}/{{tiNumber}}">{{testShortName}}</a></h3></div>
   <div class="panel-body">
 
+    {{#open}}
+    {{#allQuestionsAnswered}}
+    <p>You have answered every question. Return to Assessment overview to finish the exam.</p>
```
Comment (mwest1066) [inline frontend/require/RetryExamTestSidebarView.html]: I wonder whether this is noticeable enough? We could add a [`text-danger`](http://getbootstrap.com/css/#helper-classes-colors). Or even a [dismissible alert](http://getbootstrap.com/components/#alerts-dismissible) at the top of the page (in the top-level `#error` div?). I'm not sure exactly how visible this needs to be.

**Comment by mwest1066** [comment]
This will also resolve #190, I think.

**Comment by kevinwang** [comment]
I'm not sure about make a top level alert as that complicates data flow. How about changing the background color and text color of the div?

**Comment by tnip** [comment]
for ux purposes i'd love the bold to be on one line and the nonbold to be on another?

**Comment by kevinwang** [comment]
I agree.

**Comment by tnip** [comment]
I'm also inclined to do s/overview/Overview, but that might be out of scope.

**Comment by kevinwang** [comment]
Currently all buttons are in sentence case, but it's worth discussing.

**Comment by mwest1066** [comment]
I fear that I'm deep into bike-shedding territory at this point, but how about "to grade or finish the exam".

On capitalization I will make everything worse by proposing a third alternative with an added article, namely "Return to the assessment overview". My argument is that this is a regular sentence and we are referring to "assessment overview" as a specific noun, which is uncapitalized. :-)

I'm really completely agnostic on both of the above issues, however, so I'll leave it to @kevinwang to make the final call. If you are happy as is then just tell me and I'll merge it.

**Comment by mwest1066** [comment]
By the way, I *really* like the use of background color to highlight this message, and green is the perfect color. I now feel stupid for having suggested danger/red in the first place :-)

**Comment by kevinwang** [comment]
Sorry for the delay! I feel like "to grade or finish" would be technically correct, but a bit too verbose for a user in the typical use case.

I also think button text should be as brief as possible, and so it is fine as is.

**Pull request merged**